content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md

```diff
@@ -37,7 +37,19 @@ by providing the following flags to the kube-apiserver:
 1. Create an egress configuration file such as `admin/konnectivity/egress-selector-configuration.yaml`.
 1. Set the `--egress-selector-config-file` flag of the API Server to the path of
 your API Server egress configuration file.
-
+1. If you use UDS connection, add volumes config to the kube-apiserver:
+   ```
+    volumeMounts:
```

Comment:
This still looks incomplete and confusing..

Reply:
It works at local.

Reply:
Yes. It works on your machine. But when you are writing it as a document, you will assume the reader has no proper context. It's like copying some incomplete complete code snippet from C program and sharing it with others. Hey, this works on my machine:

```
     printf("%d", result);
   }
   check_result();
```

The first line in my example has no context. You have no idea if it is from a conditional block or a loop.
Now check your YAML.

Reply:
Done.